addons/overheating/ACE_Settings.hpp

```diff
@@ -52,4 +83,19 @@ class ACE_Settings {
         description = CSTRING(unJamFailChance_description);
         sliderSettings[] = {0, 1, 0.1, 2};
     };
+    class GVAR(cookoff) {
+        category = CSTRING(DisplayName);
+        typeName = "BOOL";
+        value = 1;
+        displayName = CSTRING(cookoff_displayName);
+        description = CSTRING(cookoff_description);
+    };
+    class GVAR(cookoffCoef) {
+        category = CSTRING(DisplayName);
+        typeName = "SCALAR";
+        value = 1;
+        displayName = CSTRING(cookoffCoef_displayName);
+        description = CSTRING(cookoffCoef_description);
+        sliderSettings[] = {0.01, 5, 1, 2};
```

Comment:
Merge this setting with above? `0 = disabled`? Might not be as clear to the user though.
@mharis001 can maybe comment as the master of friendly Arma UX.

Reply:
They could be combined, it'd mean a few more >0 checks but that's not too expensive.

Reply:
I am going to merge it

Reply:
I think in most cases combining settings that depend on each other into a single one is better. For this case, I think it makes sense and the disabled behaviour could also be described in the tooltip.